src/regions/AliveRegion.tsx

```diff
@@ -28,13 +38,20 @@ export const AliveRegion = (
 
   return observer(({ item, ...rest }: RegionComponentProps) => {
     const canRender = options?.renderHidden || !item.hidden;
+    const shouldNotUsePortal = isFF(FF_DBLCLICK_DELAY) || options?.shouldNotUsePortal;
```

Comment:
`options.shouldNotUsePortal` is set to true only in one region (Brush) and the value is `isFF(FF_DBLCLICK_DELAY)`, so it can be just removed.